lxd/project/permissions.go

```diff
@@ -1050,3 +1053,43 @@ var aggregateLimitConfigValuePrinters = map[string]func(int64) string{
 		return units.GetByteSizeString(limit, 1)
 	},
 }
+
+// FilterUsedBy filters a UsedBy list based on project access
+func FilterUsedBy(r *http.Request, entries []string) []string {
+	// Shortcut for admins and non-RBAC environments.
+	if rbac.UserIsAdmin(r) {
+		return entries
+	}
+
+	// Filter the entries.
+	usedBy := []string{}
+	for _, entry := range entries {
+		projectName := Default
+
+		// Try to parse the query part of the URL.
+		fields := strings.Split(entry, "?")
```

Comment:
@stgraber I wrote a little demo of using `url.Parse()` (which also embeds the use of `url.ParseQuery` internally) to avoid manually extracting the query string:

https://play.golang.org/p/cGtTIxQcTn4